app/grandchallenge/workstations/models.py

```diff
@@ -728,3 +739,79 @@ class FeedbackUserObjectPermission(UserObjectPermissionBase):
 
 class FeedbackGroupObjectPermission(GroupObjectPermissionBase):
     content_object = models.ForeignKey(Feedback, on_delete=models.CASCADE)
+
+
+class SessionCost(UUIDModel):
+    session = models.OneToOneField(
```

Comment:
I think `models.OneToOneField` is only a soft restriction in Django. Can we add a database constraint to ensure that there is only one `SessionCost` per `Session`?

Reply:
Okay! I thought that we hit that problem earlier but I must be mistaken.